Gemfile

```diff
@@ -6,7 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "3.2.2"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.2.1"
```

Comment:
Given how much these minor releases tend to break everything, I wonder if we can prevent Dependabot from trying to bump these automatically?